jboss-eap-standalone-openshift/scripts/ssh_linux.md

```diff
@@ -0,0 +1,22 @@
+# Red Hat OpenShift Container Platform on Azure
```

Comment:
move this and the other MD files to the root or a readme folder

Reply:
done

Reply:
I don't see it in the latest PR